docs/sources/http_api/dashboard.md

```diff
@@ -541,3 +541,97 @@ Content-Length: 97
   "status": "multiple-slugs-exists"
 }
 ```
+
+## Trim input dashboard
```

Comment:
Would move this section above the deprecated section. 

Again, should we refer to trim or name this Remove default values in dashboard?

Reply:
you are right, remove default values in dashboard would sounds better.